src/amp-story-player/amp-story-player-impl.js

```diff
@@ -349,6 +350,93 @@ export class AmpStoryPlayer {
     this.isLaidOut_ = true;
   }
 
+  /**
+   * Shows the story provided by the URL in the player.
+   * @param {string} storyUrl
+   */
+  show(storyUrl) {
```

Comment:
As a high-level question: should we sanitize the URL in some way?  Fragment params and query params should probably not be treated as different, right?

Reply:
For now we will only match the URLs as they were provided by the publisher on the player when defining the `<a>` tags. I'll add a TODO for the sanitization you mention.